readthedocs/projects/views/private.py

```diff
@@ -1131,7 +1131,7 @@ def get_context_data(self, **kwargs):
 
     def _is_enabled(self, project):
         """Should we show traffic analytics for this project?"""
-        return project.has_feature(Feature.STORE_PAGEVIEWS)
+        return True
```

Comment:
Is this method overwritten in commercial taking into account the plan?

Reply:
yep